test/e2e/generate_systemd_test.go

```diff
@@ -423,6 +423,20 @@ var _ = Describe("Podman generate systemd", func() {
 
 	})
 
+	It("podman generate systemd --container-prefix ''", func() {
```

Comment:
@vrothberg PTAL at the tests added.

Reply:
I ran the test using  `GOPATH=~/go go test -v test/e2e/libpod_suite_test.go test/e2e/common_test.go test/e2e/config.go test/e2e/config_amd64.go test/e2e/generate_systemd_test.go`

and I did not see any failure. So I pushed those changed. Hopefully there will not be any failures in CI. The following is the summary of the output of command.

```
Ran 32 of 32 Specs in 429.065 seconds
SUCCESS! -- 32 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestLibpod (429.07s)
PASS
ok  	command-line-arguments	429.114s
```